atlas/src/views/ChannelView/ChannelView.style.tsx

```diff
@@ -19,10 +20,14 @@ export const TitleSection = styled.div`
   display: flex;
   align-items: center;
   padding-top: ${theme.sizes.b10}px;
+  @media (max-width: ${theme.breakpoints.mobile}) {
```

Comment:
I see we both made improvements to that view, that's probably my bad. I added a couple more things though I think so let's get this merged and I'll rebase

Reply:
Let's leave this as is then and I'll leave it up to you to rebase / add your changes on top.